conda/install.py

```diff
@@ -575,7 +575,8 @@ def read_no_link(info_dir):
 # Should this be an API function?
 def symlink_conda(prefix, root_dir, shell=None):
     # do not symlink root env - this clobbers activate incorrectly.
-    if normpath(prefix) == normpath(sys.prefix):
+    # prefix should always be longer than, or outside the root dir.
+    if normpath(prefix) in normpath(root_dir):
```

Comment:
Seeing problems at conda-forge, likely due to this: https://github.com/conda-forge/conda-execute-feedstock/pull/8

The root prefix activate script is getting clobbered (judging by what happens locally on my computer.)

@ajspeck I'm going to fix this now, in the name of urgency.  I'll reference this comment to give you credit for your suggestion.